package.json

```diff
@@ -195,7 +195,7 @@
     "newrelic-naming-rules": "./bin/test-naming-rules.js"
   },
   "dependencies": {
-    "@grpc/grpc-js": "^1.9.4",
+    "@grpc/grpc-js": "^1.12.2",
```

Comment:
I thought we needed something recently released, but it turns out that was only for the client portion. Eh, worth setting the minimum anyway.